server/routes/admin/articles/articles.js

```diff
@@ -26,26 +26,34 @@ router.get('/add', (req, res, next) => {
 });
 
 router.post('/add', function (req, res, next) {
-    const query = req.body;
-    if (req.files) {
-        let articleImage = req.files.articleImage,
-            filename = articleImage.name;
-        articleImage.mv('../server/public/images/' + filename, function (err) {
-            if (err) {
-                console.log(err)
-                res.send(' error accurd');
-            } else {
-                console.log('done');
+    let query = req.body;        
+
+        if (Object.keys(req.files).length > 0) {
+            query = {
+                title: req.body.title,
```

Comment:
you don't need to set the whole query object again .. you just need to change `query.articleImage = req.files.articleIname.name` - the rest of the properties are already set in line 29